add several TS011F manufacturers (fixes #1099,#1096,#1078,#1069,#1036,#1006,#1019)

evervondel opened the pull request:
looked at all open TS022F device request (inculding mine 1099) and added them all

fixes #1099 
fixes #1096 
fixes #1078 
fixes #1069 
fixes #1036 
fixes #1006 
fixes #1019

objt-ev committed:
add several TS011F manufacturers

siepiee commented:
> looked at all open TS022F device request (inculding mine 1099) and added them all
> 
> fixes #1099 fixes #1096 fixes #1078 fixes #1069 fixes #1036 fixes #1006 fixes #1019

added them as they are supported now or Johan still had to look at it?

evervondel commented:
in this PR I have added them..

Op vr 20 dec 2024 19:56 schreef siepiee ***@***.***>:

> looked at all open TS022F device request (inculding mine 1099) and added
> them all
>
> fixes #1099 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1099>
> fixes #1096 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1096>
> fixes #1078 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1078>
> fixes #1069 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1069>
> fixes #1036 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1036>
> fixes #1006 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1006>
> fixes #1019 <https://github.com/JohanBendz/com.tuya.zigbee/issues/1019>
>
> added them as they are supported now or Johan still had to look at it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JohanBendz/com.tuya.zigbee/pull/1100#issuecomment-2557553771>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEVNO3Z52EBHMA66NQZ2EJD2GRR7NAVCNFSM6AAAAABT7AQ2NGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNJXGU2TGNZXGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

JohanBendz commented:
Thank you so much @evervondel, much appreciated.
I have added your collected information in the next update and closed these issues.

siepiee commented:
Any idea when this new update will be ?

dlnraja commented:
Hi, Check the actively maintained fork: https://github.com/dlnraja/com.tuya.zigbee Forum: https://community.homey.app/t/app-pro-universal-tuya-zigbee-device-app-test/140352
